BUILD

```diff
@@ -16,17 +16,33 @@ envoy_cc_library(
     repository = "@envoy",
     deps = [
         "//source/extensions/filters/http/aws_lambda:aws_lambda_filter_config_lib",
+        "//source/extensions/transformers/aws_lambda:api_gateway_transformer_lib",
         "//source/extensions/filters/http/nats/streaming:nats_streaming_filter_config_lib",
         "//source/extensions/filters/http/transformation:transformation_filter_config_lib",
     ],
 )
 
+envoy_cc_library(
+    name = "envoy_all_clusters_lib",
+    repository = "@envoy",
+    deps = [
+        "@envoy//source/extensions/clusters/aggregate:cluster",
+        "@envoy//source/extensions/clusters/dynamic_forward_proxy:cluster",
+        "@envoy//source/extensions/clusters/eds:eds_lib",
+        "@envoy//source/extensions/clusters/logical_dns:logical_dns_cluster_lib",
+        "@envoy//source/extensions/clusters/original_dst:original_dst_cluster_lib",
+        "@envoy//source/extensions/clusters/static:static_cluster_lib",
+        "@envoy//source/extensions/clusters/strict_dns:strict_dns_cluster_lib",
+    ],
+)
+
```

Comment:
why do we need to add these?

Reply:
I was receiving errors when running gloo that these extensions were not registered and found that upstream moved them out of common and made them extensions in 1.25. [Here is an example PR](https://github.com/envoyproxy/envoy/commit/1d60a116413a0422b2df50e5f6ef8b553caba53b) of one of the cluster types.